python/ray/data/tests/test_csv.py

```diff
@@ -67,7 +67,9 @@ def test_csv_read_partitioning(ray_start_regular_shared, tmp_path):
         ),
     ],
 )
-def test_csv_read(ray_start_regular_shared, fs, data_path, endpoint_url):
+def test_csv_read(
+    ray_start_regular_shared, fs, data_path, endpoint_url, target_max_block_size_none
```

Comment:
why do we want to set it to None for these csv tests?

Reply:
The fixture seems to always set the target block size to None. 
I guess we want to parameterize this config - testing both default and None, right?